server/views/pages/request/visitRequestsListing.njk

```diff
@@ -79,9 +79,13 @@
         </p>
 
         <p>
-          Requests need to be checked at least
-          <span data-test="check-before-days">{{ checkBeforeDays }} {{ "day" | pluralise(checkBeforeDays) }}</span>
-          before the visit date. If not, they will be automatically rejected.
+          Requests need to be checked 
+          {% if checkBeforeDays === 0 %}
+            <span data-test="check-before-days">before the visit date.</span>
+          {% else %}
+            <span data-test="check-before-days">at least {{ checkBeforeDays }} {{ "day" | pluralise(checkBeforeDays) }} before the visit date.</span>
```

Comment:
```suggestion
            <span data-test="check-before-days">at least {{ checkBeforeDays }} {{ "full day" | pluralise(checkBeforeDays) }} before the visit date.</span>
```

The text in the ticket is "1 **full** day" / "2 **full** days". I haven't checked but I _think_ this suggested change will work with the `pluralise()` filter.